subunit.go

```diff
@@ -68,26 +70,32 @@ type StreamResultToBytes struct {
 	Output io.Writer
 }
 
-type packet struct {
-	testID string
-	status string
+// Event is a status or a file attachment event.
+type Event struct {
```

Comment:
Scratch that, I misunderstood that this TestId is about the "client" test, not the package ones